man/systemd.network.xml

```diff
@@ -1715,6 +1715,13 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>RequestOptions=</varname></term>
+          <listitem>
+            <para>A whitespace-separated list of integers in the range 1–254.</para>
```

Comment:
This should say what this does. (The same applies also to the description of the other RequestOptions= setting for DHCPv4.)